types/types.go

```diff
@@ -15,6 +19,50 @@ const (
 	TraceLevel
 )
 
+func (l Level) String() string {
+	switch l {
+	case FatalLevel:
+		return "FATAL"
+	case ErrorLevel:
+		return "ERROR"
+	case WarnLevel:
+		return "WARN"
+	case InfoLevel:
+		return "INFO"
+	case DebugLevel:
+		return "DEBUG"
+	case TraceLevel:
+		return "TRACE"
+	default:
+		return "invalid"
+	}
+}
+
+func (l Level) Type() string {
+	return "log level"
+}
+
+func (l *Level) Set(v string) error {
+	switch strings.ToLower(v) {
+	case "fatal":
+		*l = FatalLevel
+	case "error":
+		*l = ErrorLevel
+	case "warn":
+		*l = WarnLevel
+	case "info":
+		*l = InfoLevel
+	case "debug":
+		*l = DebugLevel
+	case "trace":
+		*l = TraceLevel
+	default:
+		return errors.New("invalid log level")
```

Comment:
This error should be a static declaration at the top

Reply:
Fixed in https://github.com/loopholelabs/logging/pull/8/commits/4ab33ce91bb9689d03d006227ca8969260d84413